packages/mdc-line-ripple/README.md

```diff
@@ -1,15 +1,14 @@
 <!--docs:
-title: "Text Field Bottom Line"
+title: "Line Ripple"
 layout: detail
 section: components
-excerpt: "The bottom line indicates where to enter text, displayed below the label"
-iconId: text_field
-path: /catalog/input-controls/text-field/bottom-line/
+excerpt: "The line ripple is used to indicate certain information, such as a line of text or a selected element"
```

Comment:
The user specified text is highlighted above the line ripple.

How about that?